app/src/main/java/com/ricardoeihara/calculaflex/LoginActivity.kt

```diff
@@ -33,6 +37,7 @@ import kotlinx.android.synthetic.main.activity_login.*
                 inputLoginPassword.text.toString()
             ).addOnCompleteListener {
                 if (it.isSuccessful) {
+                    sendDataToAnalytics()
                     goToHome()
```

Comment:
You deleted this function right? Is this intended?

Reply:
@rhiroyuki No, it still exists. I deleted a commented chunk of code from this file where there was this function too.